### PR TITLE
New: [NA] Max items open control [ED-10520]

### PR DESCRIPTION
**Pull request by davseve**

### CODE REVIEW

File: modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js [hunk 1]
```diff
@@ -62,7 +62,14 @@ export default class NestedAccordion extends Base {
 		event.preventDefault();
 
 		const accordionItem = event.currentTarget.parentElement,
-			accordionContent = accordionItem.querySelector( '.e-n-accordion-item > .e-con' );
```
Comment (davseve) [inline modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js:65]: This change is not related to this PR,
Jus replacing a hard-coded selector with a predefined var we have.

**Review by Barcov — APPROVED** [review]

**Review by louiswol94 — APPROVED** [review]
